core/handler/tournament-handler.go

```diff
@@ -8,15 +8,25 @@ import (
 	"github.com/kimbellG/kerror/kegrpc"
 	ttgrpc "github.com/kimbellG/tournament/core/handler/grpc"
 	"github.com/kimbellG/tournament/core/models"
+	log "github.com/sirupsen/logrus"
 	"google.golang.org/protobuf/types/known/emptypb"
 )
 
 func (sh *ServiceHandler) CreateTournament(ctx context.Context, r *ttgrpc.CreateTournamentRequest) (*ttgrpc.CreateTournamentResponse, error) {
+	createLog := log.WithFields(log.Fields{
+		"action":  "create tournament",
+		"request": r,
+	})
+	createLog.Info("received create tournament request")
+
 	id, err := sh.tournamentController.Create(ctx, tournamentFromProto(r))
 	if err != nil {
+		kerror.ErrorLog(createLog, err, "failed to create tournament")
```

Comment:
I'm not sure why you log in kerror package. Can you explain?

I would rather just call something like logrus.WithFields(logrus.Fields{}).Error("bla bla")